snippets/getImages.md

```diff
@@ -0,0 +1,17 @@
+### getImages
+
+Fetches all images from within an element and puts them into an array
+
+Use `Element.prototype.getElementsByTagName()` to fetch all `<img>` elements inside `Element`, `Array.prototype.map()` to map every `src` attribute of their respective `<img>` element. `New Set()` creates a `Set` of unique images which is then converted into an `Array` with the `...` operator.
```

Comment:
It's just `Element.getElementsByTagName()` as `Element` **does not inherit** from `Element.prototype`. See the Array prototype [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/prototype) in comparison to the Element [docs](https://developer.mozilla.org/en-US/docs/Web/API/Element).

Reply:
okay, after researching for a while, I think you're right. My fault, sorry!